src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
```

Comment:
- Kindly specify the object because the state may contain other object categories so it will lead you to an error 👍👌
```
initialState: {
    books: [],
  },
```

Reply:
Thanks for the advice